src/merkle/tmr.rs

```diff
@@ -55,6 +55,101 @@ impl Tmr {
         0x06, 0xb2, 0x2b, 0xbe, 0x08, 0x8b, 0xb1, 0xaf,
     ]));
 
+    /// The TMRs of the types TWO^(2^n) for small values of n
```

Comment:
In the future, we can also cross this with by creating bindings with these values as computed in `precomputed.h`.